public/app/features/admin/state/reducers.ts

```diff
@@ -146,12 +146,13 @@ export const userListAdminSlice = createSlice({
   initialState: initialUserListAdminState,
   reducers: {
     usersFetched: (state, action: PayloadAction<UsersFetched>) => {
-      const { totalCount, perPage, ...rest } = action.payload;
+      const { totalCount, perPage, users, ...rest } = action.payload;
```

Comment:
Is the problem in PMM only or also in upstream?

Reply:
The code is still the same. But it comes as null from the backend, so I'm trying to figure if it still returns it that way in latest

Reply:
Just found it's fixed in the latest version, their API no longer returns null